docs/source/author/writing_guide.rst

```diff
@@ -0,0 +1,425 @@
+Writing guide
+##############################################################################
+
+This page covers:
+
+- Markdown syntax and links to Kordac processor docs
+
+The majority of our text content is written in Markdown, and we also developed
+a program called Kordac to allow authors to include complex HTML elements with
+simple text tags.
+
+We chose Markdown as it has a great balance between simplicity of writing for
+authors, and clear structure when converting to HTML.
+
+If you already know Markdown syntax, please remember the following project
+preferences (for consistency and readability):
+
+- Use asterisks (``*``) for emphasis, instead or underscores.
+- Use minuses (``-``) for unordered lists.
+- No HTML within text files, we use Kordac text tags to add iframes,
+  images, videos, etc.
+
+Below is a basic guide to syntax for Markdown and Kordac text tags.
+When viewing Kordac documentation for a tag, the top of the page will detail
+how to use the tag in a basic example.
+Some text tags also have required and/or optional tag parameters for further
+configuration.
+We link to Kordac documentation when listing their tags to prevent our
+documentation becoming out of date with Kordac updates.
+
```

Comment:
This intro is kinda hard to read. For an author, all I need is "this page will teach you what syntax we use" and maybe an example paragraph that uses a heading, image, video, panel (just three or four different tags) so I can see how these are actually used in text. I don't really need to know that Kordac exists or why we chose markdown.